src/solvers/uniswap/models.ts

```diff
@@ -0,0 +1,26 @@
+import { Address } from "@flashbots/suave-viem";
+
+
+export class Quote {
+    /** The token to sell */
+    public sell_token: Address;
+    /** The token to buy */
+    public buy_token: Address;
+    /** The minimum amount of the sell token the route must handle */
+    public sell_amount: bigint;
+    /** The maximum amount of the sell token the route must handle */
+    public buy_amount: bigint;
+    /** The L1 block number for which the route must be valid */
+    
+    constructor(
+        sell_token: Address,
+        buy_token: Address,
+        sell_amount: bigint,
+        buy_amount: bigint,
+    ) {
+        this.sell_token = sell_token;
+        this.buy_token = buy_token;
+        this.sell_amount = sell_amount;
+        this.buy_amount = buy_amount;
+    }
+}
```

Comment:
I don't mind having this, but sometimes interfaces are nicer (more concise):
```suggestion
export interface Quote {
    /** The token to sell */
    public sell_token: Address;
    /** The token to buy */
    public buy_token: Address;
    /** The minimum amount of the sell token the route must handle */
    public sell_amount: bigint;
    /** The maximum amount of the sell token the route must handle */
    public buy_amount: bigint;
}
```